tensorboard/webapp/metrics/views/right_pane/scalar_column_editor/scalar_column_editor_component.ng.html

```diff
@@ -15,20 +15,33 @@
 limitations under the License.
 -->
 <div>
```

Comment:
I don't think you need to keep this  div. It can be deleted and all its children elements moved to the top level. You still have a virtual root element, which you can refer to as :host in the css if you need to apply any styling (which you do need to do if you follow some of my other suggestions).